api/server/server.go

```diff
@@ -433,6 +434,10 @@ func (s *Server) bindHandlers(ctx context.Context) {
 	})
 }
 
+func (s *Server) Datastore() models.Datastore {
```

Comment:
can't quite figure out the use for this, it just looks out of place now next to the MQ / LogStore / etc. since this will be referenced still on the Server struct and point to the same thing, this just disallows modifying the Datastore on the Server struct? not adding thread safety or anything for initialization. anyway, not sure why this is required in this patch, it seems like we should pick one way or another for everything exported on the server and not mix and match.

Reply:
https://github.com/fnproject/fn/pull/610/files#diff-8d588bd150fe648500c167137fcc0b65R42

Reply:
hmm, ok. seems like we could have kept both method and exported `Datastore` (for uniformity in our front-end among members of `Server`), but it's a quick find & replace -- we just need to implement the additional method to satisfy ExtServer